game/forms.py

```
@@ -1,8 +1,18 @@
 class CustomUserCreationForm(UserCreationForm):
     email = forms.EmailField(required=True)
 
     class Meta(UserCreationForm.Meta):
         fields = UserCreationForm.Meta.fields + ('email',)
+
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+        if email and User.objects.filter(email__iexact=email).exists():
+            raise forms.ValidationError(
+                'An account with this email already exists. Please log in instead.'
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)